plugins/mcreator-localization/lang/texts.properties

```diff
@@ -776,6 +776,7 @@ blockly.block.block_fluidtank_fill=Fill %4 fluid to block at x: %1 y: %2 z: %3 t
 blockly.block.block_fluidtank_get=Get level of tank %4 of block at x: %1 y: %2 z: %3 on side %5
 blockly.block.block_fluidtank_get_max=Get capacity of tank %4 of block at x: %1 y: %2 z: %3 on side %5
 blockly.block.block_fluidtank_tanks=Get number of tanks of block at x: %1 y: %2 z: %3 on side %4
+blockly.block.block_registry_name=Get registry name (Block ID) of %1
```

Comment:
I'd add "or AIR if no such block", same for the item procedure, and with PIG for the entity one, to specify what happens if the entry isn't found

Reply:
But the entity one returns "minecraft:empty", not pig? Why would I make it return pig?

Reply:
Are you sure it does? Calling `ForgeRegistries.ENTITY_TYPES.getDefaultKey()` returns the pig resource location 

(Also please don't mark conversations as resolved, it's up to the reviewers!)

Reply:
Hmmm I will have to test that again then

> (Also please don't mark conversations as resolved, it's up to the reviewers!)

Will do, didn't know how that worked 👀

Reply:
(As it turns out, getting a null entity intead instantly crashes, so I will change that to check for null, and if null return default key)

Reply:
Updated accordingly